salt/modules/lxc.py

```diff
@@ -727,10 +781,10 @@ def _get_base(**kwargs):
     kw_overrides = copy.deepcopy(kwargs)
 
     def select(key, default=None):
-        kw_overrides_match = kw_overrides.pop(key, None)
+        kw_overrides_match = kw_overrides.pop(key, _marker)
         profile_match = profile.pop(key, default)
```

Comment:
ha no, cet, ok. ill be there if i ve no customer emergency as it will be on my worktime.